client/src/components/Form/FormDisplay.vue

```diff
@@ -127,6 +127,12 @@ export default {
     },
     created() {
         this.onCloneInputs();
+        // build flat formData that is ready to be submitted
```

Comment:
This is looks good to me and works well, minor suggestion is to place this into a reusable function e.g. `this.formData = this.getParams();` and reuse it below in line 169.